src/Storage/RegressionTests/adls_setaclresusive.ps1

```diff
@@ -44,13 +44,13 @@ BeforeAll {
     Add-AzAccount -ServicePrincipal -Tenant $globalNode.tenantId -SubscriptionId $globalNode.subscriptionId -Credential $cred 
 
     $resourceGroupName = $globalNode.resourceGroupName
-    $storageAccountKey = (Get-AzStorageAccountKey -ResourceGroupName $resourceGroupName -Name $testNode.accountName)[0].Value
+    $storageAccountKey = $testNode.accountKey
 
     $ctx = New-AzStorageContext  $testNode.accountName -StorageAccountKey $storageAccountKey
     $ctx2 = New-AzStorageContext  $testNode.accountName
 
     $filesystemName = "adlstest2"
-    $localSrcFile = "C:\temp\testfile_1K_0" #The file needs to exist before tests, and should be 512 bytes aligned
+    $localSrcFile = ".\data\testfile_1024K_0" #The file needs to exist before tests, and should be 512 bytes aligned
```

Comment:
Why change it to 1024K?

Reply:
Sorry had a typo here when changing the file path. Changed it back